tox.ini

```diff
@@ -19,8 +19,10 @@ passenv=SSH_AUTH_SOCK ASYNC_TEST_TIMEOUT HOME
 [testenv:pep8]
 deps=
     -rrequirements.dev.txt
-commands = flake8 src tests tests_common
+commands = sh -c 'flake8 src tests tests_common | tee flake8-report.txt'
```

Comment:
```suggestion
commands = flake8 --output-file flake8-report.txt --tee src tests tests_common
```

Reply:
Oh, very interesting, I didn't know about that.